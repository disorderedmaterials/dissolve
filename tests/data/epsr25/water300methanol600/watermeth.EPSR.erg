 -0.189182E+02  0.494219E+01  0.230666E-02  0.000000E+00  0.999999E-01  0.911491E-03  0.160907E-02  0.000000E+00  0.000000E+00  0.000000E+00
