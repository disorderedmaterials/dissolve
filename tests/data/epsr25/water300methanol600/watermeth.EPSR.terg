 -0.755227E+01  0.000000E+00 -0.755227E+01
 OW  OW 
  0.426167E+02  0.000000E+00  0.426167E+02
 OW  HW 
 -0.876011E+02  0.000000E+00 -0.876011E+02
 OW  CT 
  0.938922E+01  0.000000E+00  0.938922E+01
 OW  HC 
 -0.284484E+02  0.000000E+00 -0.284484E+02
 OW  OH 
  0.457217E+02  0.000000E+00  0.457217E+02
 OW  HO 
 -0.292284E+02  0.000000E+00 -0.292284E+02
 HW  HW 
  0.422448E+02  0.000000E+00  0.422448E+02
 HW  CT 
 -0.100245E+02  0.000000E+00 -0.100245E+02
 HW  HC 
  0.287671E+02  0.000000E+00  0.287671E+02
 HW  OH 
 -0.481947E+02  0.000000E+00 -0.481947E+02
 HW  HO 
  0.296874E+02  0.000000E+00  0.296874E+02
 CT  CT 
  0.781660E-01  0.000000E+00  0.781660E-01
 CT  HC 
 -0.460015E+01  0.000000E+00 -0.460015E+01
 CT  OH 
  0.608172E+01  0.000000E+00  0.608172E+01
 CT  HO 
 -0.443464E+01  0.000000E+00 -0.443464E+01
 HC  HC 
  0.670655E+01  0.000000E+00  0.670655E+01
 HC  OH 
 -0.208897E+02  0.000000E+00 -0.208897E+02
 HC  HO 
  0.128916E+02  0.000000E+00  0.128916E+02
 OH  OH 
  0.156910E+02  0.000000E+00  0.156910E+02
 OH  HO 
 -0.203119E+02  0.000000E+00 -0.203119E+02
 HO  HO 
  0.630537E+01  0.000000E+00  0.630537E+01
