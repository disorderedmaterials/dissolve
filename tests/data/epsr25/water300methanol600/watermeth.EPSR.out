
  Current date: 11:01:53 on 17-07-2019
  EPSR version 25: 2016-02-24      

  Trial no.     0      Iteration no.     1      No. of configurations in sum     1
  Chi-square  0.230666E-02 R-space coeff.:  0.911491E-03 Abs. empirical energy:   0.000000E+00
  Quality factor  0.160907E-02 Control level, baseline and threshold:  0.000000E+00  0.000000E+00  0.500000E+00
  Pair energies: repulsive -0.189181E+02 harmonic  0.110436E+02 per molecule
  Tethered energy:  0.000000E+00 per molecule
  Three-body energy:  0.000000E+00 per molecule
  Three-body pressure:  0.000000E+00 kbar
  No. of .erg lines:       0 Average gradient of quality factor:  0.000000E+00
  Sizefactor    1.00000 Number density  0.10000E+00
  Net charge =   -0.00001
  R-factor for data file  1:  0.2629E-02
  R-factor for data file  2:  0.1401E-02
  R-factor for data file  3:  0.6924E-03
  R-factor for data file  4:  0.6924E-03
  R-factor for data file  5:  0.2033E-02
  R-factor for data file  6:  0.5421E-03
  R-factor for data file  7:  0.3672E-02
  R-factor for data file  8:  0.6792E-02
  Noise in average g(r)  0.520884E+03
  Intermolecular energy and U_pol -0.189182E+02  0.000000E+00 kJ/mole
  Intermolecular pressure  0.494219E+01 kbar
  Average dipole moment and std.  0.161597E+01  0.601063E+00 D
  Av. intermolecular energy -0.189182E+02+- 0.000000E+00 kJ/mole
  Av. intermolecular pressure  0.494219E+01+- 0.000000E+00 kbar
  Number of rejects in MOLFIT:-
       0       0       0       0
       0       0       0       0
